schedule/yam/agama_encrypted_unattended.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
let's name the file for the test case, which will be `sles_resize_partitions`, to 'resize_partitions_unattended.yaml'

Reply:
updated.